src/views/common/base.njk

```diff
@@ -47,14 +49,17 @@
       .analyticsGa4
       .pageViewTracker
       .trackOnPageLoad({
-        statusCode: 200, englishPageTitle: `english version of the title`, // Replace with the actual page title
-        taxonomy_level1: 'tes tax 1', // Replace with the actual taxonomy level 1
-        taxonomy_level2: 'tes tax 2' // Replace with the actual taxonomy level 2
+        statusCode: '{{statusCode}}', // Access status code
+        englishPageTitle: '{{englishPageTitle}}',
+        taxonomy_level1: '{{ taxonomyLevel1 }}', // Access taxonomy level 1
+        taxonomy_level2: '{{ taxonomyLevel2 }}', // Access taxonomy level 2
       });
 
     window
       .DI
       .cookieBannerInit("localhost");
+
+    console.log(window.dataLayer);
```

Comment:
Don't forget to remove this console log when you will finish to work on your branch.